android/lib/src/main/java/com/swmansion/gesturehandler/TapGestureHandler.java

```diff
@@ -68,27 +88,74 @@ private void endTap() {
     }
   }
 
+  private static float getLastPointer(MotionEvent event, boolean isX) {
```

Comment:
Is this the same code that we have in `PanGestureHandler`? If so maybe worth extracting into some "utils" class. Please also include the comment I've put on `PanGestureHandler.getsLastPointerX` that explains what its for